documentation/docs/getting-started/installation.md

```diff
@@ -69,6 +69,9 @@ s3:
   # kubectl create secret generic securecodebox-s3-credentials --from-literal=accesskey="******" --from-literal=secretkey="******"
   keySecret: securecodebox-s3-credentials
 ```
+:::note
+If you prefer to use AWS IAM roles instead of access keys for authentication, set s3.authType to aws-irsa and s3.awsStsEndpoint should be configured as https://sts.eu-west-1.amazonaws.com in your helm values. This approach leverages IAM roles for more secure and efficient access management.
```

Comment:
IMO it would be better to use a more generic URL e.g. `https://sts.REGION.amazonaws.com` to prevent user to simply copy & paste this and accidentally use a bad region. You should also encapsulate the URL with backticks to format it as code, since Docusaurus does not allow plain URLs in markdown anymore.